### ISSUE TITLE
Accessibility: Missing/Incomplete focus outline for elements inside CMS carousels

### ISSUE THREAD

**Issue by philippsauer** [intershop/intershop-pwa#1826]
**Actual Behavior**

When focusing CMS carousel elements, no or only a partial outline is visible (neither the teaser element nor the controls provide a visible outline)

**Expected Behavior**

When focusing CMS carousel elements, the element provides a visible outline.

**Steps to Reproduce the Bug**

1. Go to https://intershoppwa-b2c.azurewebsites.net/home
2. Press tab key until you reach the teaser element
3. No outline is visible (the teaser has no visible outline)

**Environment Details**

- Desktop:
  - OS: Win 11
  - Browser: Chrome latest
  - Project Code Version: 7.0.0

**Additional Context, like Screenshots, Log File Snippets etc.**

![Image](https://github.com/user-attachments/assets/4f837060-ed71-4070-88bb-6335d34fa01f)

[AB#107390](https://dev.azure.com/intershop-com/cefd1005-00a7-4c79-927f-a16947d1b2e6/_workitems/edit/107390)
